demonstrations/tutorial_gbs.py

```diff
@@ -0,0 +1,427 @@
+r"""
+.. role:: html(raw)
+   :format: html
+
+Photonic quantum advantage with Gaussian Boson Sampling
+=======================================================
+
+.. meta::
+    :property="og:description": Using states of light carry out tasks beyond the reach of classical computers.
+    :property="og:image": https://pennylane.ai/qml/_images/qonn_thumbnail.png
+
+.. related::
+
+    tutorial_gaussian_transformation Gaussian transformation
+    qsim_beyond_classical Beyond classical computing with qsim
+    qonn Optimizing a quantum optical neural network
+    
+On the journey to large-scale fault-tolerant quantum computers, one of the first major 
+milestones is to demonstrate a quantum device carrying out tasks which are beyond the reach of 
+any classical algorithm. The Google Quantum team was the first to claim this achievement, 
+announced in their paper `Quantum supremacy using a programmable superconducting
+processor <https://www.nature.com/articles/s41586-019-1666-5>`__ [[#Arute2019]_]. Now a team led 
+by Chao-Yang Lu and Jian-Wei Pan has performed a similar feat using quantum photonics. While 
+Google's experiment performed the task of :ref:`random circuit sampling <qsim_beyond_classical>` 

```

Comment:
Probably better to do a doc link
```suggestion
Google's experiment performed the task of :doc:`random circuit sampling </demos/qsim_beyond_classical>` 
```